packages/webpack/src/utils/module-federation/get-remotes-for-host.ts

```diff
@@ -42,6 +42,8 @@ function extractRemoteProjectsFromConfig(
   const staticRemotes =
     config.remotes?.map((r) => (Array.isArray(r) ? r[0] : r)) ?? [];
   remotes.push(...staticRemotes);
+  const NPM_SCOPE_REGEX = new RegExp(/^@{1}(\w)+\/{1}(?!\/)/);
```

Comment:
nit: you can simplify this:
```suggestion
  const NPM_SCOPE_REGEX = /^@\w+\/(?!\/)/;
```